packages/embed/README.md

```diff
@@ -102,6 +102,7 @@ The options for this integration are as follows.
 | `excludedMethods`         | `null`       | An optional list of methods to be excluded.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
 | `optionLabels`            | `Object`     | An optional object map of payment option labels                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `autoSelectOption`        | `null`       | `first`, `firstStored`, `firstNonStored`, `none` - Can be used to tell Embed to automatically select either the very first payment method available, the first stored method, the first non-stored method or not to select any method at all. By default, Embed will automatically select the card method if no stored methods are active.                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
@cbetta does the description sound good?

Reply:
Yes, sounds great!